specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/2017-09-01-preview/job.json

```diff
@@ -0,0 +1,1974 @@
+          "readOnly": true,
+          "format": "date-time",
+          "description": "the completion time of the job."
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the job state. When the job is in the Ended state, refer to Result and ErrorMessage for details.",
+          "enum": [
+            "Accepted",
+            "Compiling",
+            "Ended",
+            "New",
+            "Queued",
+            "Running",
+            "Scheduling",
+            "Starting",
+            "Paused",
+            "WaitingForCapacity"
+          ],
+          "x-ms-enum": {
+            "name": "JobState",
+            "modelAsString": false
+          }
+        },
+        "result": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the result of job execution or the current result of the running job.",
+          "enum": [
+            "None",
+            "Succeeded",
+            "Cancelled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "JobResult",
+            "modelAsString": false
+          }
+        },
+        "logFolder": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the log folder path to use in the following format: adl://<accountName>.azuredatalakestore.net/system/jobservice/jobs/Usql/2016/03/13/17/18/5fe51957-93bc-4de0-8ddc-c5a4753b068b/logs/."
+        },
+        "logFilePatterns": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "the list of log file name patterns to find in the logFolder. '*' is the only matching character allowed. Example format: jobExecution*.log or *mylog*.txt"
+        },
+        "related": {
+          "$ref": "#/definitions/JobRelationshipProperties",
+          "description": "the recurring job relationship information properties."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "the key-value pairs used to add additional metadata to the job information. (Only for use internally with Scope job type.)"
+        }
+      },
+      "required": [
+        "name",
+        "type"
+      ],
+      "description": "The common Data Lake Analytics job information properties."
+    },
+    "JobInformation": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobInformationBasic"
+        }
+      ],
+      "properties": {
+        "errorMessage": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/JobErrorDetails"
+          },
+          "description": "the error message details for the job, if the job failed."
+        },
+        "stateAuditRecords": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref"
```

Comment:
The documentation of model properties MUST NOT start with the phrase "Gets or sets ..", "Gets ..", "Sets .."

Reply:
Is this referring to line 1971?

Reply:
Yes - sorry I don't know why it ends up here

Reply:
Fixed! I got rid of "Gets"